sections/Discussion.tex

```diff
@@ -36,13 +64,14 @@ \section{Discussion}\label{sec:discussion}
 Author-topic specific words are not very present and are only showing three unique words: "byder", "hører", and "børnene".
 This indicates that the Author-topic model has trouble generalizing what the author of this article (Peter Tordrup Larsen) is writing about, possibly because he has written $5002$ articles in our dataset.
 Another aspect of the Author-Topic model is that the authors writing these articles most likely do not write about just one subject, which explains why there is only three less important words marked here. 
-The Category-topic model only shows four unique words: "søndag", "leg", "klokken", and "danskere".
+The Category-topic model only shows four unique words: "søndag"(sunday), "leg"(play), "klokken"(clock), and "danskere"(danish people).
 These words are also very abstract and can be used in many different scenarios.
 
-An interesting part of this analysis is the words appearing in multiple models.
-Some of the words within this category are: "arbejdet", "jorden", "dyr", and "nordjyske".
-These words summarize the text pretty well, but it is also hard to summarize this text since it covers a wide variety of specific topics.
-Combining the results of these models might yield better topic models, but we can not conclude that based on only one article. \vejleder[inline]{why no look at more articles}
+An interesting part of this analysis is the words appearing in all models.
+Some of the words within this category are: "dansk"(danish), "millioner"(millions), "samt"(including), and "nordjyske"(north jutland).
+These words are not very descriptive, about what the article, but it is also hard to summarize this text since it covers a wide variety of specific topics.
```

Comment:
```suggestion
These words are not very representative of the content of the article, but it is also hard to summarize this text since it covers a wide variety of specific topics.
```